### DISCUSSION TITLE
mileage countdown on MCDU

### DISCUSSION THREAD

**Discussion by Tezbedz** [flybywiresim/aircraft#3458] [Help]
Do any others notice the mileage countdown on FltPln on MCDU does not countdown? Or am I doing something wrong, I am using the experimental version with FBW

**Comment by KaiBVG** [comment]
Yes, I did it here already: https://github.com/flybywiresim/a32nx/discussions/3389

**Reply by KaiBVG** [reply]
With the latest build it counts again.
